### PR TITLE
fix: add path to executable

### PR DESCRIPTION
**Pull request by josedonizetti**
make executable optional, and add path

### CODE REVIEW

File: types/api/v1beta1/event.proto [hunk 1]
```diff
@@ -50,6 +50,7 @@ message Parent  {
 
 message Executable {
     string name = 1;
```
Comment (yanivagman) [inline types/api/v1beta1/event.proto:52]: I wonder if we should make name optional as well since it can be computed from path (basename), wdyt?

**Review by yanivagman — APPROVED** [review]
LGTM

**Review by geyslan — APPROVED** [review]
LGTM

File: types/api/v1beta1/event.proto (hunk 1 above)
Comment (rafaeldtinoco) [reply types/api/v1beta1/event.proto:52]: Waiting on resolution for this one to merge (or not)

**Pull request merged**